.buildkite/pipeline.yml

```diff
@@ -128,8 +128,7 @@ steps:
         echo "--- :node: Set up Node environment"
         nvm install && nvm use
 
-        echo "--- :npm: Install Node dependencies"
-        npm ci --unsafe-perm --prefer-offline --no-audit --no-progress
+        .buildkite/commands/install-node-dependencies.sh
```

Comment:
Hey @mokagio 👋 I saw we weren't using the newly introduced `install-node-dependencies.sh` for the JS Bundle generation. Is it because we are using a docker container? I tried to add it[ but it failed](https://buildkite.com/automattic/gutenberg-mobile/builds/8881#018e3c92-b416-4506-b8a3-03983ad73c99/352-358) 😅